ceph/test_ceph.py

```diff
@@ -0,0 +1,142 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+from nose.plugins.attrib import attr
+import os
+
+# 3p
+import simplejson as json
+
+# project
+from tests.checks.common import AgentCheckTest, Fixtures
+from checks import AgentCheck
+
+
+instance = {
+    'host': 'localhost',
+    'port': 26379,
+    'password': 'datadog-is-devops-best-friend'
+}
+
+
+# NOTE: Feel free to declare multiple test classes if needed
+
+@attr(requires='ceph')
```

Comment:
This is an entirely mocked test, right? We should not spin up separate travis runs for fully mocked tests, I think. It'll really increase our testing time. If you remove this it'll just run with the default tests.